extensions/zed-recent-projects/src/lib/zedEntries.ts

```diff
@@ -162,20 +182,20 @@ export function useZedRecentWorkspaces(): ZedRecentWorkspaces {
   const query = `
     SELECT
       CASE
-        WHEN local_paths IS NOT NULL THEN 'local'
+        WHEN ssh_connection_id IS NULL THEN 'local'
         ELSE 'remote'
       END as type,
       workspace_id as id,
-      local_paths,
-      paths AS remote_paths,
+      paths as local_paths,
+      paths,
+      paths_order,
       timestamp,
       host,
       user,
       port
     FROM workspaces
-    LEFT JOIN ssh_projects ON ssh_project_id = workspaces.ssh_project_id
-    WHERE (local_paths IS NOT NULL AND paths IS NULL)
-       OR (local_paths IS NULL AND paths IS NOT NULL)
+    LEFT JOIN ssh_connections ON ssh_connection_id = ssh_connections.id
+    WHERE paths IS NOT NULL AND paths != ''
```

Comment:
In my opinion, the extension must be backward compatible with older versions; we can't simply use the new schema as it is.
Ideally, we should check which version of Zed the schema has changed from and use different queries accordingly.